src/NexusMods.App.UI/Controls/TopBar/TopBarView.axaml

```diff
@@ -60,15 +60,15 @@
                 <MenuItem Header="{x:Static resources:Language.ViewAppLogsMenuItem_Header}" x:Name="ViewAppLogsMenuItem"/>
                 <MenuItem Header="{x:Static resources:Language.GiveFeedbackMenuItem_Header}" x:Name="GiveFeedbackMenuItem"/>
             </MenuItem>
-
+            
             <MenuItem Classes="TopBarNoStyleMenuItem" x:Name="LoginMenuItem">
                 <MenuItem.Header>
-                    <Button Classes="Standard Primary" x:Name="LoginButton" Height="40">
+                    <Button Classes="Standard Primary" x:Name="LoginButton" Width="86">
                         <TextBlock Text="{x:Static resources:Language.TopBarActions__LOG_IN}"/>
```

Comment:
The last `N` is clipping. Having a fixed width here means that text will clip. Don't forget that this text is translatable, so it will appear differently depending on the language.

You should remove the fixed width.

Reply:
Good spot!

Reply:
Done, tested with a really long string